src/Raven.Server/RavenServer.cs

```diff
@@ -280,12 +280,29 @@ void ConfigureKestrel(KestrelServerOptions options)
                 if (Configuration.Server.CpuCreditsBase != null ||
                     Configuration.Server.CpuCreditsMax != null ||
                     Configuration.Server.CpuCreditsExhaustionFailoverThreshold != null ||
-                    Configuration.Server.CpuCreditsExhaustionBackgroundTasksThreshold != null)
+                    Configuration.Server.CpuCreditsExhaustionBackgroundTasksThreshold != null ||
+                    Configuration.Server.CpuCreditsExec != null ||
+                    Configuration.Server.CpuCreditsExecArguments != null)
```

Comment:
Do we have to check arguments?

Reply:
We don't *have* to... but if one of them is defined and the others not, it will give us a change to throw an appropriate error.